src/class-convertkit-api.php

```diff
@@ -173,6 +173,8 @@ public function __construct( $api_key = false, $api_secret = false, $debug = fal
 			'request_internal_server_error'               => __( 'ConvertKit API Error: Internal server error.', 'convertkit' ),
 			'request_bad_gateway'                 		  => __( 'ConvertKit API Error: Bad gateway.', 'convertkit' ),
 			
+			/* translators: API response body */
+			'response_type_null' 					  	  => __( 'ConvertKit API Error: A null response was encountered when JSON decoding %s', 'convertkit' ),
 			'response_type_unexpected' 					  => __( 'The response from the API is not of the expected type array.', 'convertkit' ),
```

Comment:
Let's update the error below to be `ConvertKit API Error: The response is not of the expected type array.`